src/App.js

```diff
@@ -1,25 +1,40 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import "./App.css";
+import allContacts from "./contacts.json"
+
+
+let fiveContacts = allContacts.slice(0,5)
+
+
 
 function App() {
```

Comment:
It's best practice to keep all the logic, all the variables inside your component. Only imports are written above